genie-web/src/main/java/com/netflix/genie/grpc/services/impl/GreeterServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *  Copyright 2016 Netflix, Inc.
```

Comment:
Wrong year.

You can configure intellij to use a template for these that have the year as a variable.